blueos_repository/consolidate.py

```diff
@@ -56,8 +73,16 @@ class Version:
     authors: List[Author]
     docs: Optional[str]
     readme: Optional[str]
-    company: Optional[Company]
+    maintainer: Optional[Maintainer]
     support: Optional[str]
+    type: ExtensionType
+    filter_tags: List[str]
```

Comment:
what are `filter_tags` ? Are they just.. `tags` ?

Reply:
Yes, but it clarifies that these are tags for filtering, as compared to the `tag` field that's the tag of the docker image.

Reply:
Maybe `category_tags` could be a clear name for people, `filter` got me pretty confuse what was the intended for it.